LocalFeeder/FeederSimulator.py

```diff
@@ -708,6 +708,46 @@ def set_properties_to_inverter(self, inverter: str, inv_control: InverterControl
         else:
             dss.Text.Command(f"{inverter}.Mode = {inv_control.mode.value}")
 
+    def set_pv_output(self, pv_system, p, q):
+        """Sets the P and Q values for a PV system in OpenDSS
+        """
+
+        max_pv = self.get_max_pv_available(pv_system)
+        pf = q / ((p**2 + q **2)**0.5)
+
+        obj_name = f"PVSystem.{pv_system}"
+        if max_pv <=0:
+            Warning("Maximum PV Value is 0")
+            obj_val = 100
+        elif p < max_pv:
+            obj_val = p/float(max_pv)
+        else:
+            obj_val = 100
+        command = [Command(obj_name=obj_name,obj_property="%PMPP",val=obj_val), Command(obj_name=obj_name,obj_property="pf",val=pf)]
+        self.change_obj(command)
+
+    def get_max_pv_available(self,pv_system):
+        dss.PVsystems.First()
+        irradiance = None
+        pmpp = None
+        while True:
+            if dss.PVsystems.Name() == pv_system:
+                print(dss.PVsystems.Name())
+                irradiance = dss.PVsystems.Irradiance()
+                pmpp = dss.PVsystems.Pmpp()
+            if not dss.PVsystems.Next() > 0:
+                break
+        if irradiance is None or pmpp is None:
+            import pdb;pdb.set_trace()
```

Comment:
This is probably not needed. P.S. Use --pdb in pytest to automatically enter debug mode on failure.